src/dbnode/integration/disk_cleanup_helpers.go

```diff
@@ -89,7 +89,10 @@ type cleanupTimesCommitLog struct {
 
 func (c *cleanupTimesCommitLog) anyExist() bool {
 	for _, t := range c.times {
-		_, index := fs.NextCommitLogsFile(c.filePathPrefix, t)
+		_, index, err := fs.NextCommitLogsFile(c.filePathPrefix, t)
+		if err != nil {
+			panic(err)
```

Comment:
Hm.. panic is a bad smell.  Is this going to get fixed by address #854 too?   

If so we should put `// https://github.com/m3db/m3/issues/854 - This requires a high priority fix to correctly handle these errors` next to each of these occurrences.  I'm still kind of wary about adding any sort of panics even as a short term fix =/